airflow/README.md

```diff
@@ -1,32 +1,32 @@
 
-Our DAGs are deployed automatically to Google Cloud Composer when a PR is merged into the `main` branch; see the section on deployment below for more information.
+Our DAGs, plugins, and changes to [the Airflow image we use for local testing](./Dockerfile) are deployed automatically to Google Cloud Composer when a PR is merged into the `main` branch; see the section on deployment below for more information. Images used by some of our Airflow DAGs (i.e. those which are configured with PodOperators) are deployed separately, often also automatically upon merged changes to the relevant Dockerfiles in the repository. System configuration for worker count, environment variables, and overrides of Airflow configs are deployed via the [Composer web console](https://console.cloud.google.com/composer/environments?project=cal-itp-data-infra), not via an automated process, but additional dependencies that we add to the standard Composer-managed Airflow install (listed in [requirements.txt](./requirements.txt)) _are_ deployed automatically upon merged changes.
 
 ## Structure
 
 The DAGs for this project are stored and version controlled in the `dags` folder. Each DAG has its own `README` with further information about its specific purpose and considerations. We use [gusty](https://github.com/pipeline-tools/gusty) to simplify DAG management.
 
 Each DAG folder contains a [`METADATA.yml` file](https://github.com/pipeline-tools/gusty#metadata) that contains overall DAG settings, including the DAG's schedule (if any).
 
-The logs are stored locally in the `logs` folder. You should be unable to add files here but it is gitkeep'ed so that it is avaliable when testing and debugging.
+The logs are stored locally in the `logs` folder. You should be unable to add files here but the folder utilizes [a .gitkeep file](https://stackoverflow.com/a/7229996) so that it is consistently avaliable when testing and debugging.
 
 Finally, Airflow plugins can be found in `plugins`; this includes general utility functions as well as custom [operator](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/operators.html) definitions.
 
-## Developing Locally
+## Testing Changes
 
-This project is developed using Docker and docker-compose. Before getting started, please make sure you have [installed Docker on your system](https://docs.docker.com/get-docker/).
+This project is developed using Docker and docker-compose. Before getting started, please make sure you have [installed Docker on your system](https://docs.docker.com/get-docker/). Docker will need to be running at the time you run any `docker-compose` commands from the console.
 
-First, if you're on linux, you'll need to make sure that the UID and GID of the container match, to do so, run
+To test any changes you've made to DAGs, operators, etc., you'll need to make sure that the UID and GID of the container match. To do so, run
```

Comment:
Resolved in [a503dd4](https://github.com/cal-itp/data-infra/pull/3156/commits/a503dd496b546a9fd8a0f3f4872b8d1b8bf8be16) - I was gonna cleave this away in a later phase, but it's probably best to do it now since I expended work updating text that didn't need to be there to begin with.